SPECS/netplan/disable-broken-tests.patch

```diff
@@ -1,9 +1,20 @@
+From 727a90437e38e70a9ab8d160fa46fc022a79e577 Mon Sep 17 00:00:00 2001
+From: Sean Dougherty <sdougherty@microsoft.com>
+Date: Tue, 5 Nov 2024 22:41:09 +0000
+Subject: [PATCH] Temporarily disabling broken test suite due to version
+ mismatches between pytest-cov and python3-coverage in 3.0. Originally added
+ by Francisco Huelsz Prince.
+
+---
+ meson.build | 10 +++++-----
+ 1 file changed, 5 insertions(+), 5 deletions(-)
+
 diff --git a/meson.build b/meson.build
-index 66cda8b..5c82f28 100644
+index b29174d..1a8f0bb 100644
 --- a/meson.build
 +++ b/meson.build
 @@ -92,11 +92,11 @@ test('legacy-tests',
-      timeout: 600,
+      timeout: 900,
```

Comment:
Why is this value increasing?

Reply:
This was updated to get the patch to apply cleanly to the new version [v1.0.1]. The upstream maintainers increased this value in the newer version.

Reply:
Ahhhh I see now. This change is not editing the parts that the .patch was originally applied for. Instead, this is a change in the surrounding context that the patch is applied to (apparently due to slow riscv64 builds...neat!). Thanks for the clarification @SeanDougherty